src/main.rs

```diff
@@ -70,10 +70,8 @@ where
 }
 
 fn main() -> anyhow::Result<()> {
-    env_logger::Builder::from_env(
-        env_logger::Env::default().default_filter_or("etf_trace=info"),
-    )
-    .init();
+    env_logger::Builder::from_env(env_logger::Env::default().default_filter_or("etf_trace=info"))
```

Comment:
This isn't generic for non-h7 targets yet, since we specifically have to handle configuring the right trace funnels. What I think needs to happen is that we need to move the `trace_funnel.rs` driver to `probe-rs` and add a new "enable_tracing" sequence for H7 targets, so all of the H7-specific configuration code would live in probe_rs.

Reply:
And yes, especially `etf.rs` and maybe everything here, should go into `probe-rs{,-cli}` etc.